core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -1554,6 +1554,8 @@ private[controller] class ControllerStats extends KafkaMetricsGroup {
 }
 
 sealed trait ControllerEvent {
+  @volatile var enqueueTimeMs: Long = -1L
```

Comment:
Does enqueueTimeMs need to be var? It seems that we can make it a val and initialize it with Time.SYSTEM.milliseconds().

Reply:
Good point. Thanks for the review! I have updated the patch as suggested.